.github/workflows/netperf.yml

```diff
@@ -27,6 +27,7 @@ on:
     - .gitmodules
     - scripts/secnetperf.ps1
     - scripts/secnetperf-helpers.psm1
+    - scripts/quic_callback.ps1
```

Comment:
Also needs to be updated above for `push`

Reply:
ACK, fixed in the next iteration.